src/renderer/components/switch/form-switcher.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import FormControlLabel, { FormControlLabelProps } from "@material-ui/core/FormControlLabel";
+import { makeStyles } from "@material-ui/styles";
+
+const useStyles = makeStyles({
+  root: {
+    margin: 0,
+    "& .MuiTypography-root": {
+      fontSize: 14,
+      fontWeight: 500,
+      flex: 1,
+      color: "var(--textColorAccent)"
+    }
+  },
+});
+
+export function FormSwitch(props: FormControlLabelProps) {
```

Comment:
What it's for? How does it work?

Reply:
`FormSwitch` wraps `Switcher` component and places some label close to it. It allows to do such a layout:
<img width="548" alt="FormSwitch" src="https://user-images.githubusercontent.com/9607060/113712014-493c3f00-96ee-11eb-88cb-af30f56ef024.png">